app/components/connectionstatus/index.js

```diff
@@ -9,22 +9,24 @@ class ConnectionStatus extends Component {
     NetInfo.isConnected.fetch()
       .done((isConnected) => {
         this.props.setIsConnected(isConnected);
-      })
+      });
   }
 
-  handleConnectionChange = (isConnected) => {
-    this.props.setIsConnected(isConnected);
-  };
-
   componentWillUnmount() {
     NetInfo.isConnected.removeEventListener('change', this.handleConnectionChange);
   }
 
+  handleConnectionChange = (isConnected) => {
+    this.props.setIsConnected(isConnected);
+  };
+
   render() {
     return (null);
   }
 }
 
-ConnectionStatus.propTypes = {};
+ConnectionStatus.propTypes = {
```

Comment:
Should the class be structured as
```javascript
class ConnectionStatus extends Component {
  static propTypes = {
    // ...
  };

  render() { /* ... */ }
}
```

so that the component will be properly statically typed.